packages/engine/state/live.ts

```diff
@@ -10,21 +12,34 @@ import type {
 import { selectors as f } from '@walless/store';
 
 import { appState } from './app';
+import {
+	collectibleActions,
+	collectibleState,
+	collectionState,
+} from './collectible';
 import { keyActions, keyState } from './key';
 import { tokenActions, tokenState } from './token';
 import { widgetActions, widgetState } from './widget';
 
 const initialize = async () => {
 	const { storage } = modules;
-	const widget = await storage.find<WidgetDocument>(f.allWidgets);
-	const allKey = await storage.find<PublicKeyDocument>(f.allKeys);
-	const allToken = await storage.find<TokenDocument>(f.allTokens);
+	const widget = await storage.find<WidgetDocument>(f.allWidgets as never);
+	const allKey = await storage.find<PublicKeyDocument>(f.allKeys as never);
+	const allToken = await storage.find<TokenDocument>(f.allTokens as never);
+	const allCollectibles = await storage.find<CollectibleDocument>(
+		f.allCollectibles as never,
+	);
```

Comment:
need to resolve types of selectors

Reply:
I think the problem is `find` method because of it's type `<never | undefined>`, whatever type you give to selectors, it'll always show errors. What do you think?